crates/bevy_pbr/src/volumetric_fog/volumetric_fog.wgsl

```diff
@@ -103,6 +103,13 @@ fn henyey_greenstein(neg_LdotV: f32) -> f32 {
     return FRAC_4_PI * (1.0 - g * g) / (denom * sqrt(denom));
 }
 
+fn simple_wrap_3(index: i32) -> i32 {
```

Comment:
Can index be >=6? If so then this is wrong, if not then I think this function should be named/commented to indicate it is special purpose.

Reply:
its only used in this file, and its only called with numbers in range 0-5. I called it simple_wrap_3, not implying its modulo, because it is not an implementation of modulo, just something that works for this specific case